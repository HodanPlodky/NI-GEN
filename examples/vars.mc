char f() {
    char c = 'a';
    return c;
}

int main() {
    int x = 1;
    int y = 2 + 5 * x;
    return y;
}
