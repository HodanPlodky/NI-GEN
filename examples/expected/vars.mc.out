thread 'main' panicked at 'not yet implemented', backend/src/inst_selection.rs:18:53
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
